torch/csrc/jit/passes/quantization/quantization_patterns.h

```diff
@@ -241,8 +241,7 @@ graph(%a_quant, %packed_params, %r_scale, %r_zero_point, %r_dtype, %stride, %pad
         return (%r_quant) )";
 
   std::string add_relu = R"(
-graph(%a_quant, %b_quant, %scale, %zero_point, %dtype):
-         %alpha = prim::Constant[value=1]()
+graph(%a_quant, %b_quant, %alpah, %scale, %zero_point, %dtype):
```

Comment:
should be `alpha`. Do we have test to catch this?

Reply:
yeah, the tests were failing and I was trying to figure out why...